index.njk

```diff
@@ -2,19 +2,54 @@
 layout: 'layouts/base.njk'
 ---
 
-<h1>
-  <span class="eyebrow">Design Sprint Guide</span>
-  Design, prototype, and test ideas quickly.
-</h1>
+<main class="tbds-app-frame" "tbds-block-stack" "tbds-block-stack--bordered">
```

Comment:
There is already a `<main>` element defined in `_includes/layouts/base.njk`, I would add the classes you need there instead, it's invalid HTML to include more than one `<main>` tag.

Reply:
```suggestion
<main class="tbds-app-frame tbds-block-stack tbds-block-stack--bordered">
```

The class attribute should only use one set of quotes, other wise everything after `tbds-app-frame` gets ignored.

Reply:
I'm not sure that the `tbds-block-stack` and `tbds-block-stack--bordered` classes are having any effect on the layout here, maybe it's best to remove them?